_posts/2024-XX-XX-quarkus-kiota.adoc

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: 'The road to generated SDKs with Kiota lands to Quarkus'
+date: 2024-XX-XX
+tags: release
+synopsis: 'Introducing a Quarkus-Kiota extension and available Java tools to work with Kiota.'
+author: andreaTP
+---
+
+== The Challenge
+:imagesdir: assets/images/posts/kiota
+
+Quarkus applications typically expose functionalities through API endpoints; here, we discuss how to easily consume those APIs and provide a smooth experience for our beloved users. While HTTP calls are easy to perform in any programming language and environment, when API complexity increases, and the number of available endpoints becomes uncountable, the need for better tools becomes apparent.
+
+image::api_monster.jpeg[API monster]
+
+=== Developer Experience Wishlist:
+
+- Fully typed programmatic APIs that make discovering and invoking HTTP endpoints safe, expressive, and elegant
```

Comment:
I know at the end you mention the existing openapi generator - but I do feel since this is on the quarkus main blog it would be good to highlight that Quarkus already has a http client approach with resteasy client that standalone or combined with the open API generator lets you have all these things....and then explain what kiota solves separately (multilanguage variations and potentially dependent on perspective a better code generation approach)